tests/plugins/fail_by_itself.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python3
+from pyln.client import Plugin
+import os
+import threading
+import time
+
+plugin = Plugin()
+
+
+class FailThread(threading.Thread):
+    def __init__(self):
+        super().__init__()
+        self.start()
+
+    def run(self):
+        time.sleep(1)
+        print("Exiting!")
+        os._exit(1)
+
+
+@plugin.init()
+def init(options, configuration, plugin):
+    FailThread()
```

Comment:
The same could be achieved by the following:

```python
def init(options, configuration, plugin):
    threading.Timer(1, lambda: os._exit(1))
```

or if the `print` is actually needed we can do this instead:

```python 
def init(options, configuration, plugin):
    def die():
        print("Exiting!")
        os._exit(1)
    threading.Timer(1.0, die)
```

:wink: